stade/core/templates/base.html

```diff
@@ -6,6 +6,19 @@
       src="https://js.sentry-cdn.com/aa507c274b254fae8fb1bd28183acf6d.min.js"
       crossorigin="anonymous"
   ></script>
+
+  {% if request.user.is_authenticated %}
+    <script>
+      Sentry.onLoad(function() {
+        Sentry.setUser({
+            'id': '{{ request.user.id }}',
+            'email': '{{ request.user.email }}',
+        });
+      });
+      Sentry.forceLoad();
```

Comment:
I don't think this is necessary just to set the user.

The docs indicate that force-loading mostly allow more breadcrumbs to be collected. Do we actually care about this?